src/status_im/chat/handlers.cljs

```diff
@@ -100,7 +100,7 @@
 (register-handler :cancel-command
   (fn [{:keys [current-chat-id] :as db} _]
     (-> db
-        (dissoc :canceled-command)
+        (assoc :canceled-command nil)
```

Comment:
`nil` doesn't say too much about possible value, so specs will be preferred for this purpose